src/main/scala/com/cognite/sdk/scala/v1/resources/events.scala

```diff
@@ -81,10 +85,18 @@ class Events[F[_]](val requestSession: RequestSession[F])
       externalIds: Seq[String],
       ignoreUnknownIds: Boolean = false
   ): F[Unit] =
-    DeleteByExternalIds.deleteByExternalIdsWithIgnoreUnknownIds(
+    DeleteByCogniteIds.deleteWithIgnoreUnknownIds(
       requestSession,
       baseUrl,
-      externalIds,
+      externalIds.map(CogniteExternalId.apply),
+      ignoreUnknownIds
+    )
+
+  override def deleteWithIgnoreUnknownIds(ids: Seq[CogniteId], ignoreUnknownIds: Boolean): F[Unit] =
```

Comment:
I think we can just call this function `delete`, the other overloads also don't have the WithIgnoreUnknownIds suffix.

Reply:
done ;)